### DISCUSSION TITLE
Reorder views

### DISCUSSION THREAD

**Discussion by patriksvensson** [community/community#4592] [Projects and Issues] [answered]
It would be great if there was a way of reordering views.
Using the image below as a reference, I would want to move `Version 0.41` next to `Everything`.

![image](https://user-images.githubusercontent.com/357872/125167354-5b6fc280-e1a0-11eb-8d09-304ff6528e36.png)

**Comment by tklein1801** [comment, marked as answer]
Dupliate of #4803

**Reply by patriksvensson** [reply]
To be fair, #4803 is a duplicate of this one 😉

**Reply by tklein1801** [reply]
Honestly, you're right. But #4803 is just the more popular one. So lets go on with his discussion 😄

**Reply by azenMatt** [reply]
The good news is we just shipped this, so you can drag and drop these tabs now!
